app/controllers/v0/rated_disabilities_discrepancies_controller.rb

```diff
@@ -55,15 +57,15 @@ def get_lh_rated_disabilities
 
       # We only want active ratings
       if response.dig('data', 'attributes', 'individual_ratings')
-        reject_deferred_ratings!(response['data']['attributes']['individual_ratings'])
+        filter_ratings_by_decision!(response['data']['attributes']['individual_ratings'])
         reject_inactive_ratings!(response['data']['attributes']['individual_ratings'])
       end
 
       response
     end
 
-    def reject_deferred_ratings!(ratings)
-      ratings.reject! { |rating| deferred?(rating) }
+    def filter_ratings_by_decision!(ratings)
+      ratings.select! { |rating| DECISION_ALLOWLIST.include?(rating) }
```

Comment:
Should this reference a field such as `rating[:decision]`?

Reply:
Yes, good call

Reply:
Fixed